src/Mesh/babylon.abstractMesh.ts

```diff
@@ -374,7 +374,7 @@
                 this.rotation = Vector3.Zero();
             }
             var rotationQuaternion: Quaternion;
-            if (!space || space === Space.LOCAL) {
+            if (!space || (space as any) === Space.LOCAL) {

```

Comment:
honestly, I don't either, I've search the compilation error on google several minutes, but it just doesn't want to compile with the === operator. and there's nothing on google about it.
I also tried a const enum, it doesn't change...